### DISCUSSION TITLE
videos highlights generator

### DISCUSSION THREAD

**Discussion by Afterpulse** [akiver/cs-demo-manager#782] [Q&A]
i did everything like in the guide
when it gets started it opens up the cs
and running the demo and when it comes to the end
its still stuck on generation in progress
i tried a few times
still like that

**Comment by akiver** [comment]
Please open a [new issue](https://github.com/akiver/cs-demo-manager/issues/new?assignees=&labels=&projects=&template=bug_report.yml) that includes a link to download the demo and steps to reproduce the problem (screenshots, videos, sequences details...)
